docs/docs/expression_language/why.ipynb

```diff
@@ -234,7 +234,7 @@
     "\n",
     "def batch_chain(topics: list) -> list:\n",
     "    with ThreadPoolExecutor(max_workers=5) as executor:\n",
-    "        return list(executor.map(invoke_chain, topics))\n",
+    "        return list(executor.map(chain.invoke, topics))\n",
```

Comment:
The LCEL examples use the function invoke_chain() so this change is not correct

Reply:
Oh, yes, thanks